README.md

```diff
@@ -2,6 +2,33 @@
 
 A migration tool does migrations for Konflux pipelines.
 
+## Run tests
+
+```bash
+python3 -m venv venv
+source ./venv/bin/activate
+python3 -m pip install -r requirements-test.txt
+tox
+```
```

Comment:
Technically, for running the tests, all I need is `tox`. What I  meant in https://github.com/konflux-ci/pipeline-migration-tool/pull/1#discussion_r1858178407 was docs for others who want to work on the code and try out the tool locally. I think setting up the venv would work better as a separate section in the README